src/csharp/Microsoft.Spark.Worker/Microsoft.Spark.Worker.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1</TargetFrameworks>
+    <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
```

Comment:
This will allow the `dotnet publish -f netcoreapp2.1 -r <runtime>` to publish without error for the Tpch and Examples projects.

Reply:
We decided not to go with this approach. You can search the issue in the old repo. 

Now that we have only one way to install worker, i think we should just remove dependencies on the worker.